README.md

```diff
@@ -1,8 +1,87 @@
 # Interoperable Immunization Data Initiative (IIDI)
 
-## One-liner summary
+## One-liner Summary
 
-TODO
+A structured implementation for **interoperable immunization data exchange**, supporting **FHIR-based APIs**, federated aggregation, and simulated cross-jurisdictional record transfers.
+
+---
+
+## Purpose
+
+The **Interoperable Immunization Data Initiative (IIDI)** provides a **development environment** for testing **data interoperability, aggregation, and transfer mechanisms** between simulated jurisdictions. The initiative focuses on **FHIR-compliant data exchange**, ensuring that immunization records remain **structured, standardized, and accessible** while respecting system independence.
+
+---
+
+## Current Development Scope (MVP)
+
+This repository implements the **minimum viable product (MVP)**, which includes:
```

Comment:
I know the biz side has reservations about the term MVP, since they want to be clear it's not going to be a product (i.e. not servicing real clients). Bit pedantic but I tend to agree, maybe we just clarify the `**minimum viable product (MVP) of the demo**`